examples/visualization/plot_study.ipynb

```diff
@@ -1,382 +1,312 @@
 {
```

Comment:
<p>I think it's good to match either <code>parameter</code> or <code>hyperparameter</code>. According to the title of this notebook, I think <code>parameter</code>  is better. How about "Visualization Parameter Relationships"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/optuna/optuna/pull/1555/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>